crates/oxc_span/src/atom.rs

```diff
@@ -59,6 +59,14 @@ impl<'a> Atom<'a> {
     }
 }
 
+impl<'old_alloc, 'new_alloc> CloneIn<'new_alloc> for Atom<'old_alloc> {
+    type Cloned = Atom<'new_alloc>;
+
+    fn clone_in(&self, alloc: &'new_alloc Allocator) -> Self::Cloned {
+        Self::Cloned::from_in(self.as_str(), alloc)
```

Comment:
I think this would be slightly clearer:

```suggestion
        Atom::from_in(self.as_str(), alloc)
```